tensorboard/program.py

```diff
@@ -556,4 +664,41 @@ def _fix_werkzeug_logging(self):
     logging.getLogger('werkzeug').setLevel(logging.NOTSET)
 
 
+class _ArgparseMonkeyPatch(object):
```

Comment:
Optional, but maybe nicer to put this in an argparse_util.py or something since it's otherwise not really relevant to program.py.